lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -518,7 +518,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :type => Array,
           :allowed_from_server => false,
           :transform => DefaultSource.method(:convert_to_regexp_list),
-          :description => 'Specify an array of Rake tasks to automatically instrument.'
+          :description => 'Specify an array of Rake tasks to automatically instrument. This configuration option converts the Array to a RegEx list. If you\'d like to allow all tasks by default, use "[.+]". No rake tasks will be instrumented unless they\'re added to this list. For more information, visit our (Rake Instrumentation)[/docs/apm/agents/ruby-agent/background-jobs/rake-instrumentation] docs.'
```

Comment:
The single backslash looks fine here. If you prefer getting rid of it, you can replace the outermost single quotes with `%q{}`

Reply:
What do you think about putting this in a HEREDOC so that the line is a little shorter?

Reply:
Zooming out to consider this entire file, I think it's rather silly that we have 255 descriptions stored in an .rb file in the first place. Perhaps we could source them from a file more suited to holding plain text. And then at that point I'd be curious to see if we could launch the agent without loading up all of that text when we don't need it.